docs/docs/user-guides/restaking-sol.md

```diff
@@ -0,0 +1,17 @@
+# How to Restake your tokens 
+
+The [MANTIS app](https://games.mantis.app/) enables you to restake your assets effortlessly using intents. You will need a Phantom wallet to interact with the application. Just prompt the token and amount you want to stake or follow the suggestions. 
```

Comment:
enables the user? do we speak in second person?

Reply:
for the guides yeah